### DISCUSSION TITLE
milo version 0.6.11 monitors the approximate limit of the number of PLC points

### DISCUSSION THREAD

**Discussion by cptjackwu** [eclipse-milo/milo#1217] [General]
Without affecting the processing speed, milo 0.6.11 version of the limit to monitor the number of PLC points is about how much, that is, more than the number of PLC, milo monitoring begins to be less, or can not monitor the change of PLC points

**Comment by kevinherron** [comment]
Any limit is going to depend on the resources of the server you are connected to. That's not something I can answer.
